acme/jose/jwk.py

```diff
@@ -120,21 +122,73 @@ def load(cls, string):
         :rtype: :class:`JWKRSA`
 
         """
-        return cls(key=util.HashableRSAKey(
-            Crypto.PublicKey.RSA.importKey(string)))
+        try:
+            key = serialization.load_pem_public_key(
```

Comment:
I haven't gotten a chance to test this, but does the private key not provide all of the functions of the private key?
My inclination would be to try to load it as a private first and then as a public.  I will try to read up on exactly what is going on here.